Source/DafnyStandardLibraries/src/Std/Unicode/UnicodeEncodingForm.dfy

```diff
@@ -127,43 +128,45 @@ abstract module Std.Unicode.UnicodeEncodingForm {
 
   /**
     * Returns the unique partition of the given code unit sequence into minimal well-formed code unit subsequences,
-    * or None if no such partition exists.
+    * or Failure(CodeUnitSeq) if no such partition exists.
```

Comment:
Can you be more specific and say it returns the suffix that couldn't be partitioned? I had to read a fair bit of other code to figure that out.